.azure/azure-build-pipeline.yml

```diff
@@ -0,0 +1,50 @@
+trigger:
+- main
+
+pr: 
+- main
+
+stages:
+  - stage: 'Test'
+    jobs:
+    - job: RunTests
+      pool:
+        vmImage: ubuntu-20.04
+      strategy:
+        matrix:
+          # Python3.7:
+          #   python.version: '3.7'
+          Python3.8:
```

Comment:
Could we change it to 3.6?
our minimum supported Python version is 3.6 and we want to verify it on Python 3.6
(people may use a Python syntax that is not available at Python 3.6).

Reply:
Yes, just added python3.6